zgw/objectenapi/src/main/kotlin/nl/nlportal/zgw/objectenapi/client/ObjectsApiClient.kt

```diff
@@ -106,6 +107,20 @@ open class ObjectsApiClient(
             .awaitSingle()
     }
 
+    suspend inline fun <reified T> createObjectWithoutCorrection(
+        objectsApiObject: CreateObjectsApiObjectRequestWithoutCorrection<T>,
+    ): ObjectsApiObject<T> {
+        return webClient()
+            .post()
+            .uri("/api/v2/objects")
+            .contentType(MediaType.APPLICATION_JSON)
+            .accept(MediaType.APPLICATION_JSON)
+            .bodyValue(objectsApiObject)
+            .retrieve()
+            .bodyToMono(object : ParameterizedTypeReference<ObjectsApiObject<T>>() {})
+            .awaitSingle()
```

Comment:
Willen wij hier al awaiten?

Reply:
Alles is in coroetines en dan is awaitSingle de default. de andere optie is block maar dan block je de threath ipv deze open houden zo als coroetines is opgebouwd. dat lijkt mij daarom de beste keuze of heb je liever block?